projects/angular-formio/grid/src/form/FormGridBody.component.ts

```diff
@@ -17,7 +17,10 @@ export class FormGridBodyComponent extends GridBodyComponent implements OnDestro
 
   load(formio: FormioPromiseService, query?: any) {
     query = query || {};
-    return formio.loadForms({ params: query })
+    return formio.loadForms(
```

Comment:
But isn't the cache generally speaking a good thing _except_ in this edge case? If I understand this correctly, this would disable browser caching for all requests of this kind. Is there a way to limit this change to only certain cases involving pagination?

Reply:
It looks to me that the etag headers are doing the right thing.  the value is changing after you submit a new form, so I don't think it's _directly_ related to the caching policy.

Also I was testing this in remote-next and couldn't reproduce.  I was also seeing the `no-cache` value set.  Let's make sure something didn't get merged or deployed by accident.